### PR TITLE
add ElectronicArts.Origin version 10.5.96.47512

### PR DESCRIPTION
**Pull request by sanyer**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/10649)

### CODE REVIEW

**Comment by sanyer** [comment]
It's failing because of MSVCP140.dll missing. Installer needs Visual C++ Redistributable for Visual Studio 2015. And weirdly enough this file exists in the installer itself.

**Comment by mddtest** [comment]
When using https://www.dm.origin.com/download/originThinSetup.exe the install appears to work fine with out the call for  MSVCP140.dl vs using https://download.dm.origin.com/origin/live/OriginSetup.exe

**Comment by OfficialEsco** [comment]
image

**Comment by sanyer** [comment]
I try again today, also both originThinSetup.exe and OriginSetup.exe

**Comment by ItzLevvie** [comment]
> When using https://www.dm.origin.com/download/originThinSetup.exe the install appears to work fine with out the call for MSVCP140.dl vs using https://download.dm.origin.com/origin/live/OriginSetup.exe

Both of them requires VC++. You likely had the dependencies already installed which is why it works fine for you, but if you try it out on Windows Sandbox then it will fail.

**Review by ItzLevvie — APPROVED** [review]

File: manifests/e/ElectronicArts/Origin/10.5.96.47512/ElectronicArts.Origin.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  - interactive
+  - silent
+  - silentWithProgress
+InstallerType: exe
+InstallerSwitches:
+  Silent: /silent
+  SilentWithProgress: /silent
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.VC++2015-2019Redist-x86
```
Comment (OfficialEsco) [inline manifests/e/ElectronicArts/Origin/10.5.96.47512/ElectronicArts.Origin.yaml:33]:
```suggestion
    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
@stephengillie can you force commit review?

**Pull request closed without merging**